dot/parachain/overseer/overseer_test.go

```diff
@@ -141,7 +157,11 @@ func TestStart2SubsytemsActivate2Different(t *testing.T) {
 }
 
 func TestStart2SubsytemsActivate2Same(t *testing.T) {
-	overseer := NewOverseer()
+	ctrl := gomock.NewController(t)
+
+	blockState := NewMockBlockState(ctrl)
+	overseer := NewOverseer(blockState)
```

Comment:
This test is failing now because of unexpected call to new mock block state:
```
=== RUN   TestStart2SubsytemsActivate2Different
    overseer.go:80: Unexpected call to *overseer.MockBlockState.GetImportedBlockNotifierChannel([]) at /home/emack/projects/ChainSafe/gossamer/dot/parachain/overseer/overseer.go:80 because: there are no expected calls of the method "GetImportedBlockNotifierChannel" for that receiver
--- FAIL: TestStart2SubsytemsActivate2Different (0.00s)
```